### PR TITLE
Allow to specify environment in sfI18nExtractTask

### PR DESCRIPTION
**Pull request by filipc**
Sf18nExtractTask did not allow to set up environment while extracting strings from project. 

When you create environment you can set different translation engine for production and development. While extracting string there was no option to specify which translation source you want to use.

This PR adds missing parameter to the task.

### CODE REVIEW

File: lib/task/i18n/sfI18nExtractTask.class.php [hunk 1]
```diff
@@ -34,6 +34,10 @@ protected function configure()
       new sfCommandArgument('culture', sfCommandArgument::REQUIRED, 'The target culture'),
     ));
 
+    $this->addOptions( array(
+      new sfCommandOption('env', null, sfCommandOption::PARAMETER_OPTIONAL, 'The environment', 'prod'),
```
Comment (j0k3r) [inline lib/task/i18n/sfI18nExtractTask.class.php]: Could you set the default to `dev` like in `sfI18nFindTask` ?

**Comment by filipc** [comment]
Default environment set to dev as asked

**Comment by j0k3r** [comment]
And finally, could you cleanup your commits? Like squashing them to avoid the merge commit you just introduced.

Thanks !

**Comment by filipc** [comment]
cleaned up :)

**Comment by j0k3r** [comment]
@filipc What happen? You force-pushed your master branch and kills the PR :confused:

**Comment by filipc** [comment]
Yes, I've force-pushed that changes. My mistake.  Please merge #92 instead.

**Pull request closed without merging**